src/lang/modifyAst/deleteSelection.ts

```diff
@@ -1,46 +1,53 @@
-import { executeAstMock } from '@src/lang/langHelpers'
-import { updateModelingState } from '@src/lang/modelingWorkflows'
-import { deleteFromSelection } from '@src/lang/modifyAst'
-import { EXECUTION_TYPE_REAL } from '@src/lib/constants'
-import type { Selection } from '@src/lib/selections'
+import { executeAstMock } from "@src/lang/langHelpers";
```

Comment:
I saw that locally with setting biome as formatter for vscode, wouldn't follow our config file :'(

Reply:
Yeah what the hell neovim is adding semicolons on save for me since then :'(